back/app/services/idea_custom_fields_service.rb

```diff
@@ -3,6 +3,14 @@ def all_fields(custom_form, options = {})
     default_fields(custom_form)
   end
 
+  def allowed_custom_field_keys custom_form
+    enabled_fields = IdeaCustomFieldsService.new.all_fields(custom_form).find_all { |f|
+      !f.code && f.enabled && !f.hidden
+    }
+    simple_keys, array_keys = enabled_fields.partition{|c| c.input_type = 'multiselect'}
```

Comment:
There is an assignment in here. I guess you meant to use `==`, because `partition` takes a boolean block.
```suggestion
    simple_keys, array_keys = enabled_fields.partition{|c| c.input_type == 'multiselect'}
```

A more general note: please do not use single letter variable names. In this case, `c` is used for a field, while on line 7 `f` is used for a field, which is not consistent and even confusing because `c` does not match anything in this method.

Reply:
Up to now, i often too the liberty of using single-letter variables when they existed for one line only, no time to get too attached to them (or bothered by them).. But I guess standards will change now ! New poeple, new rules.